tally_ho/libs/verify/quarantine_checks.py

```diff
@@ -16,6 +16,22 @@ def create_quarantine_checks():
         )
 
 
+def get_total_candidates_votes(result_form):
+    """Calculate total candidates votes for the result form
+
+    :param result_form: The result form to get candidates.
+    :returns: A Int of total candidates votes.
+    """
+    candidate_votes = []
+
+    for candidate in result_form.candidates:
```

Comment:
after looking at the [`num_votes`](https://github.com/onaio/tally-ho/blob/master/tally_ho/apps/tally/models/candidate.py#L45) function, instead of a loop there should be a single query that will do this, probably including the summation on line 32

Reply:
Let me check it out

Reply:
Done